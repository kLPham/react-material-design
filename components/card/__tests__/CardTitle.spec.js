import React from 'react';
import { shallow } from 'enzyme';
import CardTitle from '../cardTitle';

describe('<CardTitle />', () => {
    it('should render', () => {
        const title = 'this is my title';
        const cardTitle = shallow(
            <CardTitle title={title} />,
    );
        expect(cardTitle).toMatchSnapshot();
    });
    it('should render with subtitle', () => {
        const title = 'this is my title';
        const subtitle = 'this is my subtitle';
        const cardTitle = shallow(
            <CardTitle title={title} subtitle={subtitle} />,
    );
        expect(cardTitle).toMatchSnapshot();
    });
    it('should render larger', () => {
        const title = 'this is my title';
        const larger = true;
        const cardTitle = shallow(
            <CardTitle title={title} larger={larger} />,
    );
        expect(cardTitle).toMatchSnapshot();
    });
});
