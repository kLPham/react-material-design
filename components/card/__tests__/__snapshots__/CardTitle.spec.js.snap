// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardTitle /> should render 1`] = `
<section
  className="mdc-card__primary"
>
  <h1
    className="mdc-card__title"
  >
    this is my title
  </h1>
</section>
`;

exports[`<CardTitle /> should render larger 1`] = `
<section
  className="mdc-card__primary"
>
  <h1
    className="mdc-card__title mdc-card__title--large"
  >
    this is my title
  </h1>
</section>
`;

exports[`<CardTitle /> should render with subtitle 1`] = `
<section
  className="mdc-card__primary"
>
  <h1
    className="mdc-card__title"
  >
    this is my title
  </h1>
  <h2
    className="mdc-card__subtitle"
  >
    this is my subtitle
  </h2>
</section>
`;
